scala-package/pom.xml

```diff
@@ -323,11 +341,13 @@
         </executions>
       </plugin>
       <plugin>
-        <groupId>org.scala-tools</groupId>
-        <artifactId>maven-scala-plugin</artifactId>
-        <version>2.15.2</version>
+        <groupId>net.alchim31.maven</groupId>
```

Comment:
Why did this change ?

Reply:
This is switching from the older Scala building plugin to the newer one. Only the newer one is supported by scoverage and we really should be using it regardless